### DISCUSSION TITLE
bw挂两天一张没中感觉有点脸黑，请问有什么提高中票率的操作吗

### DISCUSSION THREAD

**Discussion by 199464145** [mikumifa/biliTickerBuy#395] [提问专区]
测试过了照理来说应该是能抢的呀，两天一张都没有（几乎全天蹲守），求建议，感谢，急急急。或者是什么建议检查一下的也好呀
手脚并用都没抢到TAT
大佬，菜菜，救救，呜呜

**Comment by db00719** [comment]
"上传或填入你要抢票票种的配置信息"“”“预填验证码”就是这些配置文件怎么填入啊

**Reply by SouthSeafireworks** [reply]
预填验证码不需要填，其他的看说明书

**Comment by SouthSeafireworks** [comment]
抢的哪一天的什么票种？群友也有两天一张没有的。只能是开多ip抢可以提高概率

**Reply by 199464145** [reply]
12号普票TAT

**Reply by 199464145** [reply]
多ip抢票有什么途径吗
